projects/samples/devices/worlds/.console_capabilities.wbproj

```diff
@@ -0,0 +1,10 @@
+Webots Project File version R2020a
```

Comment:
This `.wbproj` file should be configured to open the controller program in the editor window.